zerver/actions/message_send.py

```diff
@@ -158,6 +158,20 @@ def render_incoming_message(
     return rendering_result
 
 
+def get_visibility_policy_to_user_ids(stream_topic: StreamTopicTarget) -> Dict[int, Set[int]]:
+    stream_id: int = stream_topic.stream_id
```

Comment:
Why not have this as a method in the `StreamTopicTarget` class itself?

Reply:
Also see my other comment at https://github.com/zulip/zulip/pull/24236#discussion_r1105304217. It might be better to have this function return the mapping in the other way (`user_id` -> `visibility_policy`, that is)

Reply:
> Why not have this as a method in the `StreamTopicTarget` class itself?

Done